client/authority-discovery/src/tests.rs

```diff
@@ -225,18 +236,21 @@ fn new_registers_metrics() {
 }
 
 #[test]
-fn publish_ext_addresses_puts_record_on_dht() {
+fn request_addresses_of_others_triggers_dht_get_query() {
```

Comment:
The commit merges `publish_ext_addresses_puts_record_on_dht` and `handle_dht_events_with_value_found_should_call_set_priority_group` to ensure the whole cycle works. It does not touch `request_addresses_of_others_triggers_dht_get_query`.

Sadly the tests' setup code is similar, thus the git diff is a bit messy.